src/components/referrals/ImportReferralsGroupComponent.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <section class="referralsContainer">
+        <h3 class="referralsContainer__header">
+            Zaimportowane skierowania
+            <span class="numOfImport">{{
+                getPatientImportReferrals?.length
+            }}</span>
+        </h3>
+        <ul
+            v-if="getPatientImportReferrals?.length"
+            class="referralsContainer__list">
+            <import-referrals-labels
```

Comment:
for the future: good naming for v-for single element is name with single noun (e.g. refferal)